### PR TITLE
update redis set function

### PR DESCRIPTION
**Pull request by RousseauLambertLP**
Added the ability to add a yaml file or key/value combination in redis from the geomet-data-registry repo.

### CODE REVIEW

File: geomet_data_registry/store/__init__.py [hunk 1]
```diff
@@ -16,10 +16,13 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ###############################################################################
-
+import codecs
+import json
 import logging
 
 import click
+from yaml import load
```
Comment (tomkralidis) [inline geomet_data_registry/store/__init__.py]: `from yaml import Loader, load`

File: geomet_data_registry/store/__init__.py [hunk 2]
```diff
@@ -78,5 +81,36 @@ def teardown(ctx, group=None):
     click.echo('Done')
 
 
+@click.command()
+@click.pass_context
+@click.option('--key', '-k', help='key name for store')
+@click.option('--yaml-file', '-yf', 'file_',
```
Comment (tomkralidis) [inline geomet_data_registry/store/__init__.py]: `@click.option('--config, '-c', 'config',`

File: geomet_data_registry/store/__init__.py (hunk 1 above)
Comment (RousseauLambertLP) [reply geomet_data_registry/store/__init__.py]: fixed now

File: geomet_data_registry/store/__init__.py (hunk 2 above)
Comment (RousseauLambertLP) [reply geomet_data_registry/store/__init__.py]: also fixed

**Pull request merged**